gpu-operator/release-notes.rst

```diff
@@ -34,6 +34,94 @@ See the :ref:`GPU Operator Component Matrix` for a list of components included i
 
 ----
 
+.. _v24.6.0:
+
+24.6.0
+======
+
+.. _v24.6.0-new-features:
+
+New Features
+------------
+
+* Added support for the NVIDIA Data Center GPU Driver version 550.90.07.
+  Refer to the :ref:`GPU Operator Component Matrix`
+  on the platform support page.
+
+* Added support for the following software component versions:
+
+    - NVIDIA Container Toolkit v1.16.1
+    - NVIDIA Driver Manager for Kubernetes v0.6.10
+    - NVIDIA Kubernetes Device Plugin v0.16.1
+    - NVIDIA DCGM Exporter v3.3.7-3.5.0
+    - NVIDIA DCGM v3.3.7-1
+    - Node Feature Discovery v0.16.3
+    - NVIDIA GPU Feature Discovery for Kubernetes v0.16.1
+    - NVIDIA MIG Manager for Kubernetes v0.8.0
+    - NVIDIA KubeVirt GPU Device Plugin v1.2.9
+    - NVIDIA vGPU Device Manager v0.2.7
+    - NVIDIA GDS Driver v2.17.5
+    - NVIDIA Kata Manager for Kubernetes v0.2.1
+    - NVIDIA GDRCopy Driver v2.4.1-1
+
+* Added support for NVIDIA Network Operator v24.4.0.
+  Refer to :ref:`Support for GPUDirect RDMA` and :ref:`Support for GPUDirect Storage`.
+
+* Added support for using precompiled driver containers that use the 550 branch Open Kernel module drivers.
+  Specify ``--set driver.useOpenKernelModules=true --set driver.usePrecompiled=true --set driver.version=550``
+  when you install or upgrade the Operator.
+  Support remains limited to Ubuntu 22.04.
+  Refer to :doc:`precompiled-drivers` for more information.
```

Comment:
A few comments:
1) This change is not tied to a particular driver version. It is included in both the R535 and R550 variants of the container images.
2) This change is only included in precompiled container images published starting on 7/15, see https://catalog.ngc.nvidia.com/orgs/nvidia/containers/driver/tags. As we do not update previously pushed tags, all previous tags will not have this change. I am not sure if we should explicitly mention it, but here are the first tags which include this change (and all future versions will): 
```
<driver-branch>-5.15.0-116-generic-ubuntu22.04
<driver-branch>-5.15.0-1060-nvidia-ubuntu22.04
<driver-branch>-5.15.0-1063-oracle-ubuntu22.04
<driver-branch>-5.15.0-1068-azure-ubuntu22.04
<driver-branch>-5.15.0-1065-aws-ubuntu22.04
```

Reply:
I can't recall what gave me the impression it began with 550, but this is all good information and valuable in release notes.  TY!